src/SDKs/IotCentral/Management.IotCentral/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure IotCentral Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure IotCentral Resources.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
+[assembly: AssemblyFileVersion("1.1.0.0")]
```

Comment:
@dsgouda Should this be changed to `2.0.0.0` or left at `1.0.0.0` ?

Reply:
Since Nuget version is being bumped to `2.0.0` these must also be bumped to `2.0.0.0`

Reply:
Updated